packages/reporters/README.md

```diff
@@ -0,0 +1,17 @@
+# @uma/reporters
+
+This package contains two UMA reporter bot implementations: `GlobalSummaryReporter` and `SponsorReporter`.
+
+## Installing the package
+
+```bash
+yarn add @uma/reporters
+```
```

Comment:
Not sure if we've run the monitors in a while. Were you able to do a local run?

Reply:
I'm 90% sure they are actually currently broken. we should consider removing them from the repo. WDYT @nicholaspai ?